attributes/cacher.rb

```diff
@@ -0,0 +1,2 @@
+default['ssh_known_hosts']['cacher']['data_bag'] = 'server_data'
+default['ssh_known_hosts']['cacher']['data_bag_item'] = 'known_hosts'
```

Comment:
splitting up attributes files is pointless, they're all parsed no matter what recipe is in the run list.